source/Willow-Tests/TableWebViewTest.class.st

```diff
@@ -110,68 +128,75 @@ TableWebViewTest >> testRenderContentOnWithContents [
 
 	| table html |
 
-	table := TableWebView headlessDefinedBy: (Array with: (WebTableColumnRenderer untitledRenderingItemWith: [ :number | number even ])) applying: [ :theTable |  ].
+	table := TableWebViewBuilder new
+		addColumn: [ :column | column rendering: [ :number | number even ] ];
+		build.
 
 	table changeContentsTo: #(1 2 3).
 
 	html := self render: table.
 
-	self assert: html equals: '<table id="table-id1"><tbody><tr><td>false</td></tr><tr><td>true</td></tr><tr><td>false</td></tr></tbody></table>'
+	self
+		assert: html
+		equals: '<table id="table-id1"><tbody><tr><td>false</td></tr><tr><td>true</td></tr><tr><td>false</td></tr></tbody></table>'
 ]
 
 { #category : #'tests-rendering' }
 TableWebViewTest >> testRenderContentOnWithContentsAndMulticolumns [
 
 	| table html |
 
-	table := TableWebView
-		headedDefinedBy: (Array with: (WebTableMultiColumnRenderer titled: 'Common Header' renderingItemWithAll: (Array with: [ :number | number even ] with: [ :number | number odd ])))
-		applying: [ :theTable |  ].
+	table := TableWebViewBuilder new
+		addColumnTitled: 'Common Header' renderingAll: (Array with: [ :number | number even ] with: [ :number | number odd ]);
```

Comment:
This shouldn't be used, the new protocol must be applied here.

Reply:
This is using the multi-column rendering. I don't make any changes to that support yet.